tests/lib/redirectURLParser.tests.js

```diff
@@ -48,8 +48,7 @@ var tests = [
       }
     },
     result: {
-      pathname: '/de/',
-      query: {}
```

Comment:
All these tests are changed because the API is better now, and returns undefined instead of an empty object.